content/server/configuring.md

```diff
@@ -82,6 +82,16 @@ app:
     API_KEY: thisissupersecret
 ```
 
+<a name="administrator"></a>
+## Administrator
+
+After launching Skygear Server, an administrative role (`Admin`) and an
+adminitrator is created. The default username is `admin` and the default
+password is `secret`.
+
+Please remember to change the password of the administrator before your
```

Comment:
@bensonby 
Should I list out links to all SDKs or just mention that changing password via SDKs?